src/main/features/response/views/statement-of-means/priority-debts.njk

```diff
@@ -32,7 +32,7 @@
 
 
         {{ linkButton('Recalculate total monthly repayments', 'action[calculateMonthlyIncome]', 'button button-secondary calculate-monthly-income-expense') }}
```

Comment:
fix 'undefined' showing up in the browser console.

Reply:
Fixed it 

![image](https://user-images.githubusercontent.com/41957682/91037662-52661900-e601-11ea-96e7-d0aade727c1c.png)